playbook/app/pb_kits/playbook/pb_advanced_table/table_body.rb

```diff
@@ -16,6 +16,11 @@ class TableBody < Playbook::KitBase
                             default: []
       prop :collapsible_trail, type: Playbook::Props::Boolean,
                                default: true
+      prop :responsive, type: Playbook::Props::Enum,
+                        values: %w[none scroll],
+                        default: "scroll"
+      prop :is_pinned_left, type: Playbook::Props::Boolean,
```

Comment:
Do we need this separate is_pinned_left prop here and in the table_header file? this will only be applicable if responsiveness is set to scroll, correct? It feels like an unnecessary prop when you can do the logic within this .rb file if you need that variable. Creating that 'is_pinned_left' variable in the table.html just to pass it down seems unnecessary